kinetic/distribution.yaml

```diff
@@ -1617,6 +1617,22 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: kinetic-devel
     status: maintained
+  follow_waypoints:
+    doc:
+      type: git
+      url: https://github.com/danielsnider/follow_waypoints.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/danielsnider/follow_waypoints-release.git
+      version: 0.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
@danielsnider just checking, did you follow the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide the necessary access for the @ros-pull-request-builder user?